package.json

```diff
@@ -17,9 +17,9 @@
     "watch-node-feature": "export $(cat feature.env) && nodemon --watch dist/ $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
     "start-feature:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node-feature\" \"npm run watch-sass\"",
     "record-build-info": "node ./bin/record-build-info",
-    "lint": "eslint . --cache --max-warnings 200",
+    "lint": "eslint . --cache",
```

Comment:
no need for allowing warnings - we currently don't have any 🏅

Reply:
We have ours set to zero, not sure what difference that will make if we have any here?

Reply:
Ah, good spot - I saw it defaulted to -1 which I assume would be the same a 0 but in reality disables the failing.

I've explicitly set this to 0 and now it fails with any warnings